doc/config.md

```diff
@@ -1295,6 +1295,14 @@ database or default `pool_mode` is used.
 Configure a maximum for the user (i.e. all pools with the user will
 not have more than this many server connections).
 
+### client_idle_timeout
+
+Set the maximum amount of time that a client is allowed to idly connect to
+the pgbouncer instance. If set this timeout applies in addition to the
+pgbouncer level config described above.
```

Comment:
The global setting notes this:

> This should be larger than the client-side connection lifetime settings, and only used for network problems.

I guess that's not true for your usecase, otherwise you probably wouldn't need this at the user level. But that does make me curious what your usecase is.

Reply:
The usecase is that I manage a multi tenant database with users of various levels of trustworthiness. Users who connect via a client connection Pooled application are more trustworthy. Users who connect via psql or pgadmin are less trustworthy. Would like the ability to give untrustworthy users more strict settings around stuff like idle sessions or idle transactions. 

Honestly idle sessions are not the biggest deal, especially compared to idle transaction, but I figured I would put together a PR as it is a fairly easy implementation.